addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -403,7 +403,14 @@ export class HtmlField extends Component {
                 if (urgent && status(this) !== 'destroyed') {
                     await this.updateValue();
                 }
-                await savePendingImagesPromise;
+                try {
+                    await savePendingImagesPromise;
+                } catch {
```

Comment:
I think we should at least test that it's an access rights error, as we would otherwise hide any other kind of error that might happen in that rpc call to the server.

Alternatively, can't we just check that the user has write access to attachments and not make the call at all if they don't?